pcsx2/Achievements.cpp

```diff
@@ -322,7 +322,7 @@ std::string Achievements::GetGameHash(const std::string& elf_path)
 }
 
 
-void Achievements::DownloadImage(std::string url, std::string cache_filename)
+void Achievements::DownloadImage(std::string url, const std::string& cache_filename)
```

Comment:
Don't pass it by reference, instead, move it.

i.e. `auto callback = [cache_filename = std::move(cache_filename)](...`.